### PR TITLE
Podman and SELinux compatible volume declaration

### PR DESCRIPTION
**Pull request by ypconstante**
While running on Fedora 34, with podman-compose and SELinux enabled, I was getting permission errors.

[Based on this discussion](https://github.com/containers/podman-compose/issues/256), it seems compose file doesn't allow setting [SELinux label](https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label) on the long syntax, so I changed the bindings to the short format.

Errors before changes:

```
2021/11/25 00:27:29 error: open /usr/share/logstash/config/logstash.yml: permission denied
1
podman start -a docker-elk_kibana_1
internal/fs/utils.js:332
    throw err;
    ^

Error: EACCES: permission denied, open '/usr/share/kibana/config/kibana.yml'
    at Object.openSync (fs.js:497:3)
    at readFileSync (fs.js:393:35)
    at readYaml (/usr/share/kibana/node_modules/@kbn/apm-config-loader/target_node/utils/read_config.js:25:69)
    at getConfigFromFiles (/usr/share/kibana/node_modules/@kbn/apm-config-loader/target_node/utils/read_config.js:57:18)
    at loadConfiguration (/usr/share/kibana/node_modules/@kbn/apm-config-loader/target_node/config_loader.js:30:58)
    at initApm (/usr/share/kibana/node_modules/@kbn/apm-config-loader/target_node/init_apm.js:18:64)
    at module.exports (/usr/share/kibana/src/cli/apm.js:27:3)
    at Object.<anonymous> (/usr/share/kibana/src/cli/dist.js:10:17)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10) {
  errno: -13,
  syscall: 'open',
  code: 'EACCES',
  path: '/usr/share/kibana/config/kibana.yml'
}
1
Exception in thread "main" SettingsException[Failed to load settings from /usr/share/elasticsearch/config/elasticsearch.yml]; nested: AccessDeniedException[/usr/share/elasticsearch/config/elasticsearch.yml];
	at org.elasticsearch.node.InternalSettingsPreparer.prepareEnvironment(InternalSettingsPreparer.java:74)
	at org.elasticsearch.cli.EnvironmentAwareCommand.createEnv(EnvironmentAwareCommand.java:89)
	at org.elasticsearch.cli.EnvironmentAwareCommand.createEnv(EnvironmentAwareCommand.java:80)
	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:75)
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:114)
	at org.elasticsearch.cli.MultiCommand.execute(MultiCommand.java:95)
	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:114)
	at org.elasticsearch.cli.Command.main(Command.java:79)
	at org.elasticsearch.common.settings.KeyStoreCli.main(KeyStoreCli.java:32)
Caused by: java.nio.file.AccessDeniedException: /usr/share/elasticsearch/config/elasticsearch.yml
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:90)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:380)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:432)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:422)
	at java.base/java.nio.file.Files.newInputStream(Files.java:160)
	at org.elasticsearch.common.settings.Settings$Builder.loadFromPath(Settings.java:1082)
	at org.elasticsearch.node.InternalSettingsPreparer.prepareEnvironment(InternalSettingsPreparer.java:72)
	... 8 more
1
```

image

### CODE REVIEW

**Comment by antoineco** [comment]
@ypconstante thanks for the contribution!

Your conclusion that the long syntax doesn't support the `z` option seems valid, which is both disappointing and difficult to understand considering how old this feature is!

I'm torn as to whether we should make that change the default or not.
On one hand it'd be really nice to know that docker-elk works out of the box with distributions that have SELinux enabled.
On the other hand, switching back to the "short" syntax feels like a step back to me, because it's rather obscure to people who don't feel at ease with Docker but land here because they want an easy ELK set-up.

It's a trade-off. I'm leaning towards an _approval_, but let me ask @deviantony for a second opinion (_hoping this ping catches your attention_ 🙂).

Final note: as an alternative, we could add a paragraph to the existing _SELinux_ section of the README prompting SELinux users to label the docker-elk repo appropriately. However, the location of the `elasticsearch` named volume isn't know before the first "compose up". Docker is clever enough to apply the correct label, but what about Podman?

**Comment by ypconstante** [comment]
For the `elasticsearch` named volume, I got an error when running docker-compose with podman backend. Running docker-compose with docker or podman-compose with podman works.
[I've opened a bug on podman for this scenario.](https://github.com/containers/podman/issues/12421).

**Review by antoineco — APPROVED** [review]

**Comment by antoineco** [comment]
@ypconstante before I merge this, I just realized this change makes the SELinux section of the README obsolete/redundant: https://github.com/deviantony/docker-elk#selinux 

Would you mind removing it as part of this PR?

**Comment by ypconstante** [comment]
@antoineco done

**Comment by antoineco** [comment]
Thank you! 🙌

**Comment by deviantony** [comment]
@antoineco got my attention (a bit late tho :))

Agree with you, I liked the long syntax better but having support for more platforms out the box outweights this IMO.

**Pull request merged**
